stdsimd/arch/detect/os/x86.rs

```diff
@@ -116,6 +116,7 @@ fn detect_features() -> cache::Initializer {
 
         enable(proc_info_ecx, 0, Feature::sse3);
         enable(proc_info_ecx, 9, Feature::ssse3);
+        enable(proc_info_ecx, 13, Feature::cmpxchg16b);
```

Comment:
The short mnemonic for this is `cx16`. There is also a CPU feature `cx8` in `proc_info_edx = 8` that offers another CAS intrinsic. [clang](https://clang.llvm.org/docs/genindex.html) also calls this `cx16`.